tox.ini

```diff
@@ -9,3 +9,10 @@ deps =
 
 commands =
     py.test
+
+[testenv:docs]
+basepython=python
+changedir=docs
+deps=sphinx
+commands=
+sphinx-build -b html -d {envtmpdir}/doctrees docs {envtmpdir}/html
```

Comment:
Sounds good @skarbot. Hope you had a good flight back home. We're in the airport now ✈️